src/mongocxx/test/collection_mocked.cpp

```diff
@@ -272,7 +272,7 @@ TEST_CASE("Collection", "[collection]") {
         bool collection_create_index_called = false;
         bool success;
         bool expected_unique = true;
-        std::int32_t expected_expire_after_seconds = 500;
+        auto expected_expire_after = std::chrono::seconds(500);
```

Comment:
Use of {} vs () is more a matter of style in cases like this. I don't feel a need to change it.